doc/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# A Quickstart Guide to Proper Benchmarking with BenchExec
+
+This guide provides a brief summary of instructions to set up reliable
+benchmark measurements using BenchExec and important points to consider. It is
+meant for users who either want to set up benchmarking from scratch or already
+have a simple setup using, e.g., `time`, `timeout`, `taskset`, `ulimit`, etc.
+and will guide you how to use `runexec` as a simple but much more reliable
+"drop-in" replacement for these tools.
+
+## Guiding Example
+
+> [!IMPORTANT]
+> If your current setup looks similar to the below example (or you are thinking
+> about such a setup), we strongly recommend following this guide for a much
+> more reliable process.
+
+As an example, suppose that you want to measure the performance of your
+tool `program` with arguments `--foo` and `--bar` on the input files
+`input_1.in` to `input_9.in`. To measure the runtime of the tool, one may run
+```
+$ /usr/bin/time program --foo input_1.in
+$ /usr/bin/time program --bar input_1.in
+$ /usr/bin/time program --foo input_2.in
+...
+```
+etc. and note the results. In case resource limitations are desired (e.g. 
+limiting to 1 CPU and 60 sec of wallclock time), the calls might be
+```
+$ taskset -c 0 timeout 60s /usr/bin/time program ...
+```
+or similar.
+
+## Benchmarking with BenchExec
+
+The following steps guide you to increase the reliability and quality of
+measurements drastically by using BenchExec instead of these small
+utilities.
+
+### Step 1. Install BenchExec
+
+Depending on your distribution, the setup steps vary slightly.
+On Debian/Ubuntu and similar, you can
+```
+sudo add-apt-repository ppa:sosy-lab/benchmarking
+sudo apt update && sudo apt install benchexec
+```
+Otherwise, try the pip-based setup
+```
+pip3 install --user benchexec[systemd] coloredlogs
+```
+
+You should be able to run the command
+`python3 -m benchexec.check_cgroups` without issues. See
+[the installation guide](INSTALL.md) for further details and troubleshooting.
+
+### Step 2. Consider the Setup of Your Benchmark
+
+Consider and document the benchmarking process *beforehand*. In particular,
+think about which executions you want to measure, what resource limits should
+be placed on the benchmarked tool(s), such as CPU time, CPU count, memory, etc.
+Also consider how timeouts should be treated.
+
+For more complicated setups, please also refer to the
+[benchmarking setup guide](benchmarking.md). For example, in case you want to
+execute multiple benchmarks in parallel, think about how to deal with shared
+resources (e.g. the memory bus and CPU cache). For such cases, we recommend
+using `benchexec` instead, which takes care of managing parallel invocations.
+
+
+### Step 3. Gather Measurements using runexec
+
+Using the example from above, suppose that we want to limit the process to 60s
+wall time, 1 GB of memory, and cpu core 0. Then, simply run
```

Comment:
We already had users confusing the `--cores` setting of `runexec` with a limit on the number of cores. I would take the chance here to try and clarify this with something like "... and one CPU core (pinning the process to CPU core 0 of the system)".